LICENSE

```diff
@@ -1,212 +1,97 @@
-      incurred by, or claims asserted against, such Contributor by reason
-      of your accepting any such warranty or additional liability.
-
-   END OF TERMS AND CONDITIONS
-
-   APPENDIX: How to apply the Apache License to your work.
-
-      To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright [yyyy] [name of copyright owner]
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       https://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
-
----------------
-
-Licenses for dependency projects can be found here:
-[https://akka.io/docs/akka/snapshot/project/licenses.html]
-
----------------
-
-akka-protobuf contains the sources of Google protobuf 2.5.0 runtime support,
-moved into the source package `akka.protobuf` so as to avoid version conflicts.
-For license information see COPYING.protobuf
+Business Source License 1.1
+
+Parameters
+
+Licensor:             Lightbend, Inc.
+Licensed Work:        Akka 2.7.x
+                      The Licensed Work is (c) 2022 Lightbend Inc.
+Additional Use Grant:
+    If you develop an application using a version of Play Framework that
+    utilizes binary versions of akka-streams and its dependencies, you may
+    use such binary versions of akka-streams and its dependencies in the
+    development of your application only as they are incorporated into
+    Play Framework and solely to implement the functionality provided by
+    Play Framework; provided that, they are only used in the following way:
+    Connecting to a Play Framework websocket and/or Play Framework
+    request/response bodies for server and play-ws client.
+
+Change Date:          TBD ( 3 years after 2.7.0 release )
+
+Change License:       Apache License, Version 2.0
+
+For information about alternative licensing arrangements for the Software,
+please visit: https://lightbend.com/
+
+-----------------------------------------------------------------------------
+
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
```

Comment:
MariaDB not Lightbend?

Reply:
This copyright is for the license text for "Business Source License 1.1" itself